internal/host/hostutil/test_utils.go

```diff
@@ -122,7 +122,34 @@ func GenerateMasterInventoryWithHostnameV6(hostname string) string {
 				},
 			},
 		},
-		Memory:       &models.Memory{PhysicalBytes: hardware.GibToBytes(16)},
+		Memory:       &models.Memory{PhysicalBytes: hardware.GibToBytes(16), UsableBytes: hardware.GibToBytes(16)},
+		Hostname:     hostname,
+		SystemVendor: &models.SystemVendor{Manufacturer: "Red Hat", ProductName: "RHEL", SerialNumber: "3534"},
+		Timestamp:    1601835002,
+	}
+	b, err := json.Marshal(&inventory)
+	Expect(err).To(Not(HaveOccurred()))
+	return string(b)
+}
+
+func GenerateInventoryWithResources(cpu, memory int64, hostname string) string {
```

Comment:
Where do we use this function?

Reply:
It's being used to generate a host manifest for testing purposes in the [/internal/host/transition_test.go](https://github.com/openshift/assisted-service/blob/master/internal/host/transition_test.go#L2585) test file